src/app/_helpers/filter-conf.ts

```diff
@@ -0,0 +1,68 @@
+import { FilterExecution } from '../_models/filterExecution';
+
+export const filterConf: FilterExecution[] = [
+  {
+    label: 'workflow',
+    name: 'WORKFLOW',
+    multi: true,
+    options: [
+      { value: 'enrich' },
+      { value: 'import a' },
+      { value: 'import b' },
+      { value: 'normalise' },
+      { value: 'preview' },
+      { value: 'validate' },
+      { value: 'transform' },
```

Comment:
No, they're configuration options.

Reply:
I've moved the configuration out of _helpers and into filter-ops/filter-ops/conf.  I've also made the config generate its values automatically where it can.